src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(5);
```

Comment:
Consider defining the number of figure types as a constant, e.g., `private static final int FIGURE_COUNT = 5;`, to avoid using magic numbers.